share_extension/android/extension_post/extension_post.js

```diff
@@ -63,7 +63,7 @@ export default class ExtensionPost extends PureComponent {
         actions: PropTypes.shape({
             getTeamChannels: PropTypes.func.isRequired,
         }).isRequired,
-        channelId: PropTypes.string.isRequired,
```

Comment:
We might not have it as checked in the connected component 

https://github.com/mattermost/mattermost-mobile/blob/da32c22c568657569cecad2f504b516cc2db86b4/share_extension/android/extension_post/index.js#L26-L28